.gitignore

```diff
@@ -19,6 +19,9 @@ pl-training.yml
 *.tar
 *.sif
 
+# Tutorial files
+tutorials/hpo-workflows/data/
```

Comment:
this could be just `data/`

Reply:
If it's just `data/` will it ignore all data folders though?

Reply:
yes, do we have folders called data? If yes, leave it like that

Reply:
Actually I checked and it seems no, and also I just saw that there is already a `/data` field, which didn't catch my one because it only looks in the root folder (I assume) - so I've changed this to `data/` to catch everything